src/__tests__/__snapshots__/integration-test.ts.snap

```diff
@@ -0,0 +1,87 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`integration with apollo-server correct span nesting 1`] = `
+Object {
+  "children": Array [
+    Object {
+      "children": Array [
+        Object {
+          "children": Array [
+            Object {
+              "children": Array [],
+              "finished": true,
+              "id": 7,
+              "logs": Array [
+                Object {
+                  "result": "\\"2\\"",
+                },
+              ],
+              "name": "two",
```

Comment:
It's not very obvious from the snapshot but here we have the span `two` nested undnearth the span `one` for query 

`query {
        a {
          one
          two
        }
      }`

Reply:
This means now we would need to change the snapshot so it fails and put your new implementation on it, so the bug is fixed?

Reply:
Yeah, but I think the snapshot is almost impossible to read so I feel like it adds little value. Maybe a custom snapshot serializer